### PR TITLE
Kartikey_Assignment_22_Done_12mins

### PR DESCRIPTION
**Pull request by KartikeyDJune20**
image
image

### CODE REVIEW

File: src/kartikeyDJune20/PalindromeString.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package kartikeyDJune20;
+
+public class PalindromeString {
+
+	static void checkStringPalindrome(String input) {
+		String rev = "";
+		for (int index = input.length() - 1; index >= 0; index--) {
+			rev = rev + String.valueOf(input.charAt(index));
```
Comment (harshvegada) [inline src/kartikeyDJune20/PalindromeString.java]: Here, we don't need of String.valurOf() method as we concat the string first and then char, but if it's reverse then we will go this method. No impact on logic .

Comment (KartikeyDJune20) [reply src/kartikeyDJune20/PalindromeString.java]: yes yes.. Thanku Harsh.. it should be optimized.

Comment (KartikeyDJune20) [reply src/kartikeyDJune20/PalindromeString.java]: Changes Incorporated..

**Review by harshvegada — COMMENTED** [review]
One correction @KartikeyDJune20 
Here we are checking exact to exact string if both case match then only pass the condition 
Do not use equvalsIgnore case 
Correct the changes

File: src/kartikeyDJune20/PalindromeString.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		}
+		if (rev.equals(input))
+			System.out.println("Given string " + input + " is Palindrome");
+		else
+			System.out.println("Given string " + input + " is not Palindrome");
+	}
+
+	public static void main(String[] args) {
+		PalindromeString.checkStringPalindrome("technocredits");
+		PalindromeString.checkStringPalindrome("Radar");
```
Comment (KrishnaKTechnocredits) [inline src/kartikeyDJune20/PalindromeString.java:18]: R should be small otherwise it will say it is not palindrome :)

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**